Automattic/jetpack
# Dependencies: Remove unused dependencies

#### Changes proposed in this Pull Request:

Remove the following dependencies from `projects/plugins/jetpack/package.json`:

* json-loader
* cross-env
* @automattic/custom-colors-loader
* husky

Additionally, the root level file `postcss.config.js` references two dependencies in `/projects/plugins/jetpack/package.json`, `postcss-custom-properties` and `@automattic/calypso-color-schemes`. It's not clear to me if `postcss.config.js` is being utilized, but it would seem to make sense moving it to `/projects/plugins/jetpack` at least.

#### Jetpack product discussion

* N/A

#### Does this pull request change what data or activity we track or use?

* No

#### Testing instructions:

* After checking out branch, run `yarn distclean` and `yarn build` watching for any build errors.
* These dependencies should not be in use currently, so there is not a specific area to check for, but browsing through WP Admin Jetpack plugin pages is suggested, keeping an eye out for any errors.
* It's also suggested to try a few of the Jetpack blocks, maybe one or two with color settings.

#### Proposed changelog entry for your changes:

* N/A
